tests/test_backend.py

```diff
@@ -495,11 +494,16 @@ def test_delete_pattern_with_custom_count(self, client_mock, cache: RedisCache):
 
     @patch("django_redis.cache.RedisCache.client")
     def test_delete_pattern_with_settings_default_scan_count(
-        self, client_mock, cache: RedisCache
+        self,
+        client_mock,
+        cache: RedisCache,
+        settings: SettingsWrapper,
     ):
+        settings.DJANGO_REDIS_SCAN_ITERSIZE = 30
```

Comment:
can you use `@override_settings()` instead?

Reply:
Done!